integration-tests/docker/test_env/test_env.go

```diff
@@ -149,14 +149,15 @@ func (te *CLClusterTestEnv) StartMockAdapter() error {
 	return te.MockAdapter.StartContainer()
 }
 
-func (te *CLClusterTestEnv) StartClCluster(nodeConfig *chainlink.Config, count int, secretsConfig string, opts ...ClNodeOption) error {
+// pass config here
+func (te *CLClusterTestEnv) StartClCluster(nodeConfig *chainlink.Config, count int, secretsConfig string, testconfig *testconfig.TestConfig, opts ...ClNodeOption) error {
```

Comment:
@Tofel maybe we should not reference `testconfig.TestConfig` here but just pass `imageName, imageVersion string` or `[]ChainlinkImageConfig` to define different versions for each node?

Reply:
Why so? Soon we will want to be able to pass here custom node config (that's coming in v2).

Reply:
@Tofel I'd like to reuse `StartClCluster` in Mercury. But, I don't want to pass `testconfig.TestConfig` which contains  things like `Common`, `Automation`, `VRF`, etc. just to provide image name and image versions for the nodes in the cluster. I'd just pass `[]ChainlinkImageConfig`.

Reply:
I see, then maybe we could pass something yet different, which is a struct that will contain:
* image data
* CL node configuration
Wdyt?

That, or an interface.